src/SFA.DAS.EmploymentCheck.Functions/Application/Models/EmploymentCheck.cs

```diff
@@ -54,6 +54,7 @@ public class EmploymentCheck
         public bool? Employed { get; set; }
 
         public short? RequestCompletionStatus { get; set; }
+        public short VersionId { get; set; }
```

Comment:
Is this still being used?

Reply:
Nope, missed on removal of all the references to versioning